1Flow-web doc (#5623)
segmentio/segment-docs: src/connections/destinations/catalog/1flow-web/index.md

---
title: 1Flow Web (Actions) Destination

> id: 64dd07c1fed86b6866cd93f5

Reviewer: ```suggestion
hidden: true
```

The destination has not been deployed yet, so there is no existing id. We will keep this hidden until the integration is in Public Beta

versions:
  - name: '1Flow Web (Actions) Destination'
    link: '/docs/connections/destinations/catalog/1flow-web'

---
[1Flow](https://1flow.ai/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank"} is a leading in-app user survey and messaging platform for Mobile app and SaaS businesses.

{% include content/plan-grid.md name="actions" %}

1Flow is an easy-to-use, yet powerful in-app survey and messaging software.
Using 1Flow, you can reach users in-the-moment while they are interacting with your website or mobile app, to collect highly contextual user insights that help you improve your product offering and customer experience.

When you use the 1Flow Web (Actions) Destination, Segment loads the [1Flow SDK](https://1flow.ai/docs/install-sdk/javascript){:target="_blank"} for you. The 1Flow library enables you to track and identify user events on your website and interact with the 1Flow messenger window.

> 

Reviewer: ```suggestion
{% include content/ajs-upgrade.md %}
```

Add this snippet, which will include content on ensuring they are using a.js 2.0

## Benefits of 1Flow Web Mode (Actions) Destination

After you install both Segment and 1Flow's Segment Plugin for the web:
1. You can track events and identify users
2. Segment passes the event and user payload directly to 1Flow 
3. 1Flow then uses this information to trigger surveys and identify users

## Getting started
1. From Segment, navigate to  **Connections > Catalog**, then select **Destinations**.
2. Search for and select **1Flow  Web (Actions) Destination**.
3. Click **Configure 1Flow Web (Actions) Destination**
4. Select the web source that will send data to 1Flow web (Actions) and follow the steps to name your destination. The web source chosen must use [Analytics.js 2.0](/docs/connections/source/catalog/libraries/website/javascript).
5. On the **Settings** tab, input your 1Flow "PROJECTAPIKEY" and other destinations settings.
6. Follow the step in the Destinations Actions docs to [customizing mappings](/docs/connections/destinations/action/#customizing-mappings).
 7. Enable the destination and configured mappings.

> 

Reviewer: ```suggestion
```suggestion
{% include components/actions-fields.html %}
```

For actions destinations, this snippet will autopopulate information related to the available actions

## Supported methods

### Identify

> If you're not familiar with the Segment Specs, take a look to understand what the [Identify method](/docs/connections/spec/identify/) does. An example call would look like:

Reviewer: ```suggestion
If you're not familiar with the Segment Specs, take a look to understand what the [Identify method](/docs/connections/spec/identify/) does. Below is an example of how a Segment Identify event maps to an invocation of the 1flow SDK:
```

Clarify this, because this mapping is done automatically. Don't want to cause confusion and have them think they need to add the 1flow invocations in their codebase.

```window._1flow('identify', 'userId', {
			'name': 'Jane Doe', 
			'company': 'ACME, Inc.'
			'email': 'jane.doe@gmail.com', 
			'plan': 'Premium'
			'total_spend': 880
	});

```
When you call Segment's Identify method, it will be equivalent to `logUser` of 1Flow. 
- Segment's `userId` is `userID` in 1Flow 
- Segment's `traits` is `userDetails` in 1Flow

### Track

> If you're not familiar with the Segment Specs, take a look to understand what the [Track method](/docs/connections/spec/track/) does. An example call would look like:

Reviewer: ```suggestion
If you're not familiar with the Segment Specs, take a look to understand what the [Track method](/docs/connections/spec/track/) does. Below is an example of how a Segment Track event maps to an invocation of the 1flow SDK:
```

same here
